setup.py

```diff
@@ -39,8 +39,19 @@
       },
       install_requires=[
           'boto>=2,<3', 'boto3',
-          'jsonschema', 'timeout_decorator', 'awscli',
-          'PyYAML', 'psycopg2',
```

Comment:
See notes in INSTALL.md for why I think dropping `psycopg2` here is the right change.